packages/frontend/src/common/components/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Box, Link, Paper } from "@mui/material";
+
+interface FooterProps {
+  isTransparent?: boolean;
+}
+export default function Footer({ isTransparent = false }: FooterProps) {
+  return (
+    <Paper
+      elevation={0}
+      sx={{
+        position: "absolute",
+        bottom: 0,
+        width: "100%",
+        backgroundColor: isTransparent ? "transparent" : undefined,
+        marginBottom: "1rem",
+      }}
+    >
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          gap: 1,
+          justifyContent: "center",
+        }}
+      >
+        <Link href="/privacy" color={isTransparent ? "#000000" : undefined}>
```

Comment:
Here i am unsure how to provide the color. The default black on other components is the initial value, but a color="initial" is not allowed. So as the color is no part of the theme and the homepage is not using it, i chose to set the raw value.

Reply:
Removed it entirely, the footer is now consistent on all sites